types/browan/TBClimate/meta.json

```diff
@@ -1,23 +1,20 @@
 {
-  "name": "Temperature & Humidity Sensor, TBHH100, Tabs",
+  "name": "Tabs Temperature & Humidity Sensor, TBHH100",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed